src/main.ts

```diff
@@ -32,7 +32,8 @@ async function run(): Promise<void> {
     getArgs('--output', 'output_format'),
     getArgs('--config-file', 'config_file'),
     getArgs('--skip-resources', 'skip_resources'),
-    getArgs('--parsers', 'parsers')
+    getArgs('--parsers', 'parsers'),
+    getArgs('--tag-modules', 'tag_modules')
```

Comment:
```suggestion
    getArgs('--tag-local-modules', 'tag_local_modules')
```
looking at bridgecrewio/yor#310 shouldn't be `--tag-local-modules`?